docs/readme.md

```diff
@@ -70,6 +70,7 @@ Release Notes
 ### Upcoming Release
 - Changes
     - Analytics: Added `SetConsent()` and `GetSessionIdAsync()` APIs.
+    - Crashlytics: Add `ReportUncaughtExceptionsAsFatal` propertie and `LogExceptionAsFatal` API. Crashlytics now support on-demand fatal reports.
```

Comment:
```suggestion
    - Crashlytics: Add `ReportUncaughtExceptionsAsFatal` property and `LogExceptionAsFatal` API. With these changes, Crashlytics now supports on-demand fatal reports.
```

Reply:
Also, I don't know if the style for Unity release notes use the verb form of "Add" vs "Added"?

Reply:
From here I can see they are using `Add` a lot for new features: https://firebase.google.com/support/release-notes/unity#10.2.0

Reply:
Done for the change!

Reply:
Removed notes for support of ODF.